udata/search/fields.py

```diff
@@ -56,7 +58,20 @@ def get_value_filter(self, value):
 
 
 class TermsFacet(Facet, DSLTermsFacet):
-    pass
+
+    def add_filter(self, filter_values):
+        if not filter_values:
```

Comment:
I forgot to explain: the `must`/`should`/`shouldnot` is the default ES `AND`/`OR`/`NOT` operations and ES-DSL automatically use it with boolean operations so:
```python
q1, q2 = Q(), Q()
q1 or q2 == Q('bool', should=[q1, q2])
```